[2025-07-06 18:50:48,443] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 1.1, 10, 1, 8.0, 10, 1, 10, 1)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:50:48,451] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:50:48] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:00,273] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 1.1, 12, 0, 8.0, 10, 1, 10, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:00,276] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:00] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:03,933] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 1.1, 12, 1, 8.0, 10, 1, 10, 1)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:03,936] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:03] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:12,988] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 1.1, 12, 1, 8.0, 10, 0, 10, 1)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:12,989] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:12] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:16,331] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 1.1, 12, 0, 8.0, 10, 0, 10, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:16,336] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:16] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:29,019] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 10.2, 12, 0, 8.0, 10, 0, 10, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:29,023] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:29] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:40,532] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 10.2, 12, 0, 8.0, 10, 0, 40, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:40,535] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:40] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:51:55,965] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 10.2, 0, 0, 8.0, 10, 0, 40, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:51:55,969] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:51:55] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 18:52:48,037] WARNING predict() L186  Erro geral ao adicionar aluno: Não foi possível salvar novo aluno :/. Detalhes: (sqlite3.OperationalError) no such table: alunos
[SQL: INSERT INTO alunos ("Age", "Gender", "Academic_Level", "Country", "Avg_Daily_Usage_Hours", "Most_Used_Platform", "Affects_Academic_Performance", "Sleep_Hours_Per_Night", "Mental_Health_Score", "Relationship_Status", "Conflicts_Over_Social_Media", outcome) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)]
[parameters: (55, 1, 1, 7, 20.0, 0, 0, 8.0, 10, 0, 40, 0)]
(Background on this error at: https://sqlalche.me/e/20/e3q8) - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\app.py L186 
[2025-07-06 18:52:48,040] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 18:52:48] "[31m[1mPOST /aluno HTTP/1.1[0m" 400 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 19:12:37,457] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 19:12:37,458] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 19:12:42,676] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 19:12:42] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 19:12:42,908] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 19:12:42] "[36mGET /front/my_website/index.html HTTP/1.1[0m" 304 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 19:12:42,940] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 19:12:42] "[36mGET /front/my_website/assets/css/style.css HTTP/1.1[0m" 304 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
[2025-07-06 19:12:42,944] INFO _log() L97   127.0.0.1 - - [06/Jul/2025 19:12:42] "[36mGET /front/my_website/midia.png HTTP/1.1[0m" 304 - - call_trace=D:\Documents\addicted_v0.1.4\Projetinho\backend\venv\Lib\site-packages\werkzeug\_internal.py L97  
